docs/learningLab/introToLit/intro.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 1
+---
+
+# 1. Intro
+At its core, Lit is an attempt to decentralize [public key cryptography](https://www.cloudflare.com/learning/ssl/how-does-public-key-encryption-work/). This technology underpins all of cryptocurrency and a majority of the security infrastructure on the web today. A decentralized key management network, Lit can be used in place of centralized key custodians and other key management solutions
+
+Public key infrastructure (PKI) powers two main "buckets" of functionality: encryption and signing.
+
+### **Encryption**
+
+Encryption is the process of encoding information so that it remains hidden or inaccessible to unauthorized parties.
+
+Lit's first feature is a **decentralized access control** protocol interoperable with most EVM chains, Solana, and Cosmos. With Lit, you can use on-chain access control conditions to do 4 main things:
```

Comment:
can we simplify this description? We got rid of the 'static' vs 'dynamic' distinction in a previous doc revamp, is there a reason you think this distinction is important to include here?